app/Account/Controllers/AccountController.php

```
@@ -9,85 +9,80 @@
+     * Updates the current user account.
+     */
     public function update(UpdateAccountRequest $request): JsonResponse
     {
-        $user = Auth::guard('jwt')->user();
-
-        if ($user === null) {
-            abort(Response::HTTP_UNAUTHORIZED);
-        }
-
-        $account = $this->accountService->updateForUser($user, $request->validated());
+        $account = $this->accountService->update($request->validated());
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$account'. (undefined)

(UnusedLocalVariable)